test/jdk/java/lang/invoke/VarHandles/X-VarHandleTestAccess.java.template

```diff
@@ -506,6 +492,7 @@ public class VarHandleTestAccess$Type$ extends VarHandleBaseTest {
 #end[Bitwise]
     }
 
+
```

Comment:
Extra blank line?

Reply:
This extra blank line is in the mainline version.